Events/hirse-024-2024-01.md

```diff
@@ -0,0 +1,37 @@
+# Webinar: Best practice made easy: Deploying tools for FAIR research software development
+
+- Event Date: 2024-01-23
+- Location: Online
+- Event Website: <https://www.helmholtz-hirse.de/series/2024_01_23-seminar_24.html>
+- Organizers: Helmholtz Platform for Research Software Engineering
+
+This event is part of the HiRSE Seminar Series, a series of talks revolving around topics relevant to research software engineering and research software engineers. The seminars cover a broad range of topics from political to technical ones, and take place every 2 to 4 weeks.
```

Comment:
Could you add `https://www.helmholtz-hirse.de/series` as link for "HiRSE Seminar Series"?